checkpoint/cloudguard/README.md

```diff
@@ -0,0 +1,167 @@
+
+Refer to [values.yaml](values.yaml) for the full run-down on defaults. These are a mixture of Kubernetes and CloudGuard directives that map to environment variables.
+
+Specify each parameter using `--set key=value[,key=value]` or `--set-string key=value[,key=value]` to `helm install`. For example,
+
+```bash
+$ helm install my-release --set varname=true --set-string varname="string" checkpoint/cp-resource-management
+```
+
+Alternatively, a YAML file that specifies the values for the parameters can be provided while installing the chart. For example,
+
+```bash
+$ helm install my-release -f values.yaml checkpoint/cp-resource-management
+```
+
+> **Tip**: You can use the default [values.yaml](values.yaml)
+
+The following tables list the configurable parameters of this chart and their default values.
+
+| Parameter                                                  | Description                                                     | Default                                          |
+| ---------------------------------------------------------- | --------------------------------------------------------------- | ------------------------------------------------ |
+| `clusterID`                                                | Cluster Unique identifier in CloudGuard system                  | `CHANGEME`                                       |
+| `region`                                                   | CloudGuard region (us1, eu1 or ap1)                             | `us1`                                            |
+| `credentials.secret`                                       | CloudGuard APISecret                                            | `CHANGEME`                                       |
+| `credentials.user`                                         | CloudGuard APIID                                                | `CHANGEME`                                       |
+| `imagePullPolicy`                                          | Image pull policy                                               | `IfNotPresent`                                   |
+| `imageRegistry.url`                                        | Image registry                                                  | `quay.io`                                        |
+| `imageRegistry.org`                                        | Image registry organization                                     | `checkpoint`                                     |
+| `imageRegistry.authEnabled`                                | Whether or not Image Registry access is password-protected      | `true`                                            |
+| `imageRegistry.user`                                       | Image registry username                                         | ``                                               |
+| `imageRegistry.password`                                   | Image registry password                                         | ``                                               |
```

Comment:
I would add to the description that also a `token` can be used. It is always better practice since tokens can be revoked.